### PR TITLE
Add alert variation with description

### PR DESCRIPTION
**Pull request by madalena-campos-brisa**
Two properties were added:
description: Validates the use of description in the alert;
noRadius: Allows the removal of the alert's border-radius.

It was necessary to change the structure of the alert, but visually it is the same.

### CODE REVIEW

**Review by allan-chagas-brisa — CHANGES_REQUESTED** [review]

File: projects/ion/src/lib/alert/alert.component.spec.ts [hunk 1]
```diff
@@ -103,6 +103,17 @@ describe('AlertComponent', () => {
...
+  it('should render with description', async () => {
+    const description = 'Testing description';
+    const element = await sut({ ...defaultValue, description });
+    expect(element).toHaveTextContent(description);
+  });
+
+  it('should render without radius', async () => {
+    const element = await sut({ ...defaultValue, noRadius: true });
+    expect(element).toHaveClass('no-radius');
+  });
```
Comment (iurynogueira) [inline projects/ion/src/lib/alert/alert.component.spec.ts:115]: create a new test case to when have radius, with noRadius false.

**Review by iurynogueira — CHANGES_REQUESTED** [review]

File: projects/ion/src/lib/alert/alert.component.html [hunk 2]
```diff
@@ -1,27 +1,43 @@
...
+      data-testid="close-icon"
+      type="close"
+      *ngIf="closable"
+      (click)="closeEvent()"
+    ></ion-icon>
+  </div>
+
+  <div *ngIf="description?.length > 0" class="description">
+    {{ description }}
+  </div>
```
Comment (allan-chagas-brisa) [inline projects/ion/src/lib/alert/alert.component.html:42]:
```suggestion
  <span *ngIf="description?.length > 0" class="description">
    {{ description }}
  </span>
```
It remained a div.

**Review by allan-chagas-brisa — CHANGES_REQUESTED** [review]

File: projects/ion/src/lib/alert/alert.component.scss [hunk 3]
```diff
@@ -47,11 +58,33 @@
...
+  color: $neutral-7;
+  gap: 4px;
+
+  .message-text {
+    font-weight: 600;
+  }
+
+  .description {
+    word-break: break-all;
+    hyphens: auto;
```
Comment (allan-chagas-brisa) [inline projects/ion/src/lib/alert/alert.component.scss:74]: I think I removed these properties when submited the review, you can see on the other comment.

File: projects/ion/src/lib/alert/alert.component.scss [hunk 4]
```diff
@@ -47,11 +58,33 @@
   @include alert-style($negative-1, $negative-6);
 }
 
+.with-description {
+  border: none !important;
+  background-color: $neutral-2;
+  padding: 16px 24px !important;
```
Comment (allan-chagas-brisa) [inline projects/ion/src/lib/alert/alert.component.scss:64]: sorry, forgot to sugest the sass functions.
```suggestion
  padding: spacing(2) spacing(3) !important;
```

**Review by allan-chagas-brisa — APPROVED** [review]
Nice :)

**Review by iurynogueira — APPROVED** [review]

**Pull request merged**